store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1072,6 +1079,7 @@ func (mvcc *MVCCLevelDB) CheckTxnStatus(primaryKey []byte, lockTS, callerStartTS
 				// We *must* guarantee the invariance lock.minCommitTS >= callerStartTS + 1
 				if lock.minCommitTS < callerStartTS+1 {
 					lock.minCommitTS = callerStartTS + 1
+					action = kvrpcpb.Action_MinCommitTSPushed
```

Comment:
We need to set this action even if minCommitTS is already greater than callerStartTS

Reply:
Oh, get it!
That information is actually 'could the caller read ignore the lock' ... instead of 'minCommitTS pushed'